src/qililab/instruments/qblox/qblox_module.py

```diff
@@ -127,6 +126,18 @@ def desync_sequencers(self) -> None:
         for sequencer in self.awg_sequencers:
             self.device.sequencers[sequencer.identifier].sync_en(False)
 
+    def set_markers_override_enabled_by_port(self, value: bool, port: str):
+        """Set markers override flag ON/OFF for the sequencers associated with port."""
+        sequencers = self.get_sequencers_from_chip_port_id(chip_port_id=port)
+        for sequencer in sequencers:
+            self.device.sequencers[sequencer.identifier].marker_ovr_en(value)
+
+    def set_markers_override_value_by_port(self, value: int, port: str):
```

Comment:
why is the port necessary if we are using the values that represent all the channel indexes?

Reply:
Because each sequencer can set the markers independently, and the final value is determined by OR operation.

Reply:
aka there is no way to set the markers in the module level, the method exists on Sequencer API